### ISSUE TITLE
When updating Select2 to version 3.4.5, ui-select2 ng-model binding does not reflect changes to the $scope

### ISSUE THREAD

**Issue by aruizca** [angular-ui/ui-select2#124]
This is the main issue that I have workaround by calling the componet explicitly instead of modifying the $scope variable.

Now I have this: 

``` javascript
$('#permissions').select2('val', data.permissions);
```

where I used to have this:

```
$scope.user.permissions = data.permissions;
```

Unfortunately this have some side effects like not eliminating the selected options from the drop-down list.

**Comment by aruizca** [comment]
I am using Angular 1.0.8, select2 3.4.5 and angular-ui-select2 0.0.4

**Comment by BaconSoap** [comment]
Can you create a plunkr to demonstrate this? At the moment, the tests for this repo don't pass when running against angular 1.0.8 - there were some significant breaking changes between that and the 1.2.x releases. It should be possible to go back to angular-ui-select2 0.0.2, or a point before the 0.0.3 release, and use that, if you can't upgrade angular.

**Comment by aruizca** [comment]
You are right. Downgrading to version 0.0.2 fixes this particular problem with Angular 1.0.8, but introduces some others. I think it is fair to say that in order to use this component you better use AngularJS 1.2.x :-)
